lib/split/dashboard/views/index.erb

```diff
@@ -1,10 +1,12 @@
 <% if @experiments.any? %>
   <p class="intro">The list below contains all the registered experiments along with the number of test participants, completed and conversion rate currently in the system.</p>
 
-  <input type="text" placeholder="Begin typing to filter" id="filter" />
-  <input type="button" id="toggle-completed" value="Hide completed" />
-  <input type="button" id="toggle-active" value="Hide active" />
-  <input type="button" id="clear-filter" value="Clear filters" />
+  <div class="dashboard-controls">
+    <input type="text" placeholder="Begin typing to filter" id="filter" />
+    <input type="button" id="toggle-completed" value="Hide completed" />
+    <input type="button" id="toggle-active" value="Hide active" />
+    <input type="button" id="clear-filter" value="Clear filters" />
+  </div>
```

Comment:
Already in our code: https://github.com/clio/split/blob/master/lib/split/dashboard/views/index.erb#L4-L9